saga-web/src/main/resources/saga-frontend/README.md

```diff
@@ -0,0 +1,87 @@
+[![Build Status](https://travis-ci.org/akveo/ngx-admin.svg?branch=master)](https://travis-ci.org/akveo/ngx-admin)
```

Comment:
Update this file with saga frontend information

Reply:
@asifdxtreme  Fixed